astropy/table/serialize.py

```diff
@@ -66,7 +76,16 @@ class SerializedColumn(dict):
     Normally contains the single key ``name`` with the name of the column in the
```

Comment:
This is no longer true.

Reply:
Agreed, that was out of date already. Removed it.